test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -1142,6 +1142,37 @@ TEST_F(AtenXlaTensorTest, TestAllDimKeep) {
   }
 }
 
+TEST_F(AtenXlaTensorTest, TestAMin) {
+  torch::Tensor input =
+      torch::rand({4, 3, 4}, torch::TensorOptions(torch::kFloat));
+  int rank = input.dim();
+  for (bool keepdim : {false, true}) {
+    for (int dim = -rank; dim < rank; ++dim) {
+      auto values_indices = torch::amin(input, {dim}, /*keepdim=*/keepdim);
```

Comment:
ditto, what's the return type of `torch::amin`? If it is just `torch::tensor` we should probably spell it.

Reply:
Addressed.